web/docs/operators/transformations/top.md

```diff
@@ -5,12 +5,16 @@ Shows the most common values. The dual to [`rare`](rare.md).
 ## Synopsis
 
 ```
-top <field> [--count-field=<count-field>|-c <count-field>]
+top <n> by <field> [--count-field=<count-field>|-c <count-field>]
 ```
 
 ## Description
 
-Shows the most common values for a given field. For each field value, the result will be a singular event containing the value's count.
+Shows the most common values for a given field. For each unique value, a new event containing its count will be produced.
+
+### `<n>`
```

Comment:
For `head` and `tail` we call this argument `<limit>`, should it have that name here as well for consistency?

Applies to `rare` as well.

Reply:
Yes. 
Changed.